mods/ra/maps/drop-zone-w/weapons.yaml

```diff
@@ -2,42 +2,19 @@
 	ReloadDelay: 200
 	Range: 32c0
 	Burst: 4
-	Report: turret1.aud
 	Projectile: Bullet
 		Speed: 546
-		High: true
-		Angle: 62
 		Inaccuracy: 3c341
-		Image: 120MM
-		ContrailLength: 30
 	Warhead: SpreadDamage
```

Comment:
This is adding an additional warhead (with 100% versus everything).  I guess it wants to be `Warhead@1Dam: SpreadDamage` so that it overrides the default warhead instead?

Reply:
Whoops, I overlooked that.